src/core/client_config/lb_policies/protos/load_balancer.proto

```diff
@@ -0,0 +1,115 @@
+syntax = "proto3";
```

Comment:
Isn't it different for each repo? I'd imagine a src/proto mirroring test/proto would be reasonable, and easy for us in Java to find.